models/topics.models.js

```diff
@@ -1,7 +1,16 @@
 const db = require("../db/connection");
+const fs = require("fs/promises");
+const path = require("path");
 
-exports.fetchAllTopics = (req) => {
+exports.fetchAllTopics = () => {
   return db.query(`SELECT * FROM topics`).then((results) => {
     return results.rows;
   });
 };
+
+exports.fetchApiInfo = () => {
+  const endpointsDataPath = path.resolve(__dirname, "../endpoints.json");
```

Comment:
This is fine but it's worth noting that the model is usually for database interactions which we don't have here. We could simply send the endpoints file in the controller